src/ui/src/app/repository-new/repository-new.component.html

```diff
@@ -10,11 +10,7 @@ <h2 class="repository-new__title">Add Repository</h2>
     </mat-form-field>
     <mat-form-field class="repository-new__form__field">
       <input matInput name="repository-url" placeholder="URL" required
-        [(ngModel)]="repoAttributes.URL" (ngModelChange)="formError=null">
-    </mat-form-field>
-    <mat-form-field class="repository-new__form__field">
-      <input matInput name="repository-source" placeholder="Source"
-        [(ngModel)]="repoAttributes.source" (ngModelChange)="formError=null">
+        [(ngModel)]="repoAttributes.url" (ngModelChange)="formError=null">
```

Comment:
the current `dev-v1` branch was failing due to the change of the repository model